song/services/postService.js

```diff
@@ -0,0 +1,34 @@
+const postDao = require("../models/postDao");
+
+const postingList = async () => {
+  const postingLoad = await postDao.postingLoad();
+  return postingLoad;
+};
```

Comment:
```suggestion
const postingList = async () => {
  return postDao.postingLoad();
};
```
위에서 남겨드린 코멘트와 동일한 맥락이며 적용되는 기타 부분들도 일괄적으로 수정될 수 있게 해주세요.